src/main/java/core/Main.java

```diff
@@ -0,0 +1,16 @@
+package core;
```

Comment:
```suggestion
package com.internet.shop;
```
please put **all your code** in this package